localstack/aws/api/awslambda/__init__.py

```diff
@@ -160,10 +160,6 @@ class ProvisionedConcurrencyStatusEnum(str):
 
 class Runtime(str):
     nodejs = "nodejs"
-    nodejs4_3 = "nodejs4.3"
```

Comment:
This is a generated file, we should not modify anything here. It will get updated once it is removed from the spec.

Reply:
Fixed 👍